its/src/test/java/com/sonar/it/csharp/MetricsTest.java

```diff
@@ -33,6 +37,9 @@
 import static com.sonar.it.csharp.Tests.getMeasureAsInt;
 import static org.apache.commons.lang.StringUtils.countMatches;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.junit.Assume.assumeFalse;
+import static org.junit.Assume.assumeTrue;
```

Comment:
Thanks! Is there an option for IntelliJ (or a plugin) to automatically remove unused imports?